application/config/session.php

```diff
@@ -18,7 +18,7 @@
     |
     */
 
-    'driver' => env('SESSION_DRIVER', 'file'),
+    'driver' => env('SESSION_DRIVER', 'redis'),
```

Comment:
just a thought; instead of hard coding it the app config, wouldn't it be better to configure this via the `.env` and have corresponding example in the `.env.example`?

Reply:
While we're here, I would probably switch out the `queue` driver to `redis` also.

Reply:
Yes! And you can totally override everything in the configs including the `SESSION_DRIVER` in the .env file. I'm just just setting sane defaults for people who might be running this thing but not familiar with larval. I will update the queue and make redis also the default in the .env.example file.